docs/building.md

```diff
@@ -2,21 +2,6 @@
 
 The process for building Atmosphère is similar to building Fusée Gelée payloads and other Switch apps.
 
-## Linux
+In order to build Atmosphère you must have devkitPro installed on your computer. You can find instructions on how to install and setup devkitPro on various OSes here.
```

Comment:
This is very wrong. devkitPro is *not* software. devkitPro is an organization that produces tools. You need to install devkitARM and devkitA64.